services/video_chess_indexer/board.py

```diff
@@ -15,6 +22,23 @@ def create(self):
                 row.append(Square())
             self.squares.append(row)
 
+    def create_game_start(self, flipped):
+        starting_indexes = [0,1,6,7]
+
+        if flipped is True:
+            starting_indexes = [7,6,1,0]
```

Comment:
Can just have one list of starting_indexes and use .reversed(), this is super minor so up to you.